Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -244,6 +244,8 @@ bool CruRawReader::parseDigitHCHeaders(int hcid)
   // mHBFoffset32 is the current offset into the current buffer,
   //
   mDigitHCHeader.word = mHBFPayload[mHBFoffset32++];
+  mPreTriggerPhase = INVALIDPRETRIGGERPHASE; // set to 15 (invalid) incase there is no half header with the pretriggerphase, which is optional
+  // valid values are 0,3,6,9 or 2,5,8,11 (see asci art in header of explanation of why.
```

Comment:
this comment could be removed, so that we have all the explanations in one place (Digit.h) and don't forget synching it..

Reply:
ja